neo/SmartContract/Contract.py

```diff
@@ -114,7 +114,7 @@ def CreateMultiSigContract(publicKeyHash, m, publicKeys):
 
         pk = [ECDSA.decode_secp256r1(p).G for p in publicKeys]
         return Contract(Contract.CreateMultiSigRedeemScript(m, pk),
-                        bytearray(b'\x00\x00\x00'),
+                        bytearray(m),
```

Comment:
The `ParameterList` argument was fixed to 3 signatures, while it should be dynamic based on how many signatures the contract requires ([C# source](https://github.com/neo-project/neo/blob/879952a8259347db3f04211502efebe1dec202cd/neo/SmartContract/Contract.cs#L69)). A ContractParameterType of type `signature` is `0x00`, so we can simply re-use the `bytearray` constructor.